protocol.html

```diff
@@ -334,6 +334,8 @@ <h3 property="schema:name">Definitions</h3>
 
                   <p>A <dfn class="dfn-paneled" data-dfn-type="dfn" id="solid-app">Solid app</dfn> is an application that reads or writes data from one or more <a data-link-type="dfn" href="#data-pod" id="ref-for-data-pod">data pods</a>.</p>
 
+                  <p>An <dfn class="dfn-paneled" data-dfn-type="dfn" id="owner">owner</dfn> is a person or a social entity identified by a URI that has implicit control of a data pod.</p>
```

Comment:
I think we're conflating the definition of an owner with the abilities arising from it (control).

Not perfect, but:

```suggestion
                  <p>An <dfn class="dfn-paneled" data-dfn-type="dfn" id="owner">owner</dfn> of a data pod is a person or other agent identified by a URI that is considered responsible for its storage. Such responsibility arises, for instance, when someone registers a pod, control its domain name, etc. Owners implicitly have control over all data in the pod.</p>
```

Reply:
So far the notion of "owner" was deemed to be in the scope of a data pod / storage and was not in the layer of controlling the domain name, URI ownership etc. The suggestion above conflicts with the Note. Of course nothing stops us from defining the owner to beyond the storage. I think Solid Protocol's terms should stay within or thereabouts the features it enables.

I like the take on "considered" and "responsible", so did some shuffling around and came up with this: https://github.com/solid/specification/pull/264/commits/6ba032d7556f42d2323a339adfeb03965d973024 . WDYT?

I also tried to keep it somewhat compatible with https://dpvcg.github.io/dpv/#hasDataController . "Owner" should decide on how data can be ultimately processed in a storage, and also be held accountable.

Reply:
Looks good, just added some small nits.